apis/v1alpha1/gatewayclass_types.go

```diff
@@ -85,10 +85,25 @@ type GatewayClassSpec struct {
 type GatewayClassConditionType string
 
 const (
-	// GatewayClassConditionStatusInvalidParameters indicates the
-	// validity of the Parameters set for a given controller. This
-	// will initially start off as "Unknown".
-	GatewayClassConditionStatusInvalidParameters GatewayClassConditionType = "InvalidParameters"
+	// GatewayClassConditionStatusAdmitted indicates that the GatewayClass
+	// has been accepted by the controller set in the `spec.controller`
+	// field.
+	//
+	// It defaults to Unknown, and MUST be set by a controller when it sees
+	// a GatewayClass using its controller string.
```

Comment:
This should be updated to match the new default value.

Reply:
🤦